vars/createArgusTestRun.groovy

```diff
@@ -2,14 +2,15 @@
 
 def call(Map params) {
     def test_config = groovy.json.JsonOutput.toJson(params.test_config)
+    def cloud_provider = getCloudProviderFromBackend(params.backend)
     sh """#!/bin/bash
         set -xe
 
         echo "Creating Argus test run ..."
         if [[ -n "${params.requested_by_user}" ]] ; then
             export BUILD_USER_REQUESTED_BY=${params.requested_by_user}
         fi
-        export SCT_CLUSTER_BACKEND="${params.backend}"
+        export SCT_CLUSTER_BACKEND="${cloud_provider}"
```

Comment:
Why are we changing this ? It doesn't seem to be related to this change set

Reply:
This is my mistake.
Initial implementation for this PR used another word in Jenkins parameters, to indicate docker backend. And that word was not known in sdcm.SCTConfiguration.defaults_config_files map, so some logic was added to work around that.
But at the end simple 'docker' wording for indicating backend in Jenkins is working well for existing sct_config logic.
So this change is not needed, reverting it back.

Reply:
so please revert it